apps/studio/components/ui/Forms/FormFieldWrapper.tsx

```diff
@@ -0,0 +1,53 @@
+import {
+  FormField_Shadcn_,
+  FormItem_Shadcn_,
+  FormLabel_Shadcn_,
+  FormDescription_Shadcn_,
+  FormControl_Shadcn_,
+  FormMessage_Shadcn_,
+} from 'ui'
+import { Control } from 'react-hook-form'
+import { ReactNode } from 'react'
+
+interface FormFieldWrapperProps {
+  control: Control<any>
+  name: string
+  label: string
+  description?: string | ReactNode
+  orientation?: 'horizontal' | 'vertical'
+  children: (field: any) => ReactNode
+}
+
+export function FormFieldWrapper({
+  control,
+  name,
+  label,
+  description,
+  orientation = 'horizontal',
+  children,
+}: FormFieldWrapperProps) {
+  return (
+    <FormField_Shadcn_
+      control={control}
+      name={name}
+      render={({ field }) => (
+        <FormItem_Shadcn_
```

Comment:
I foresee potentially confusing with this file as we've got a lot of form related components atm
so this probably needs some JSDocs to explain when to use it

also should we use FormItemLayout instead of FormItem_Shadcn? (You can sort of see the confusion haha 😅 )
ref of usage [here](https://github.com/supabase/supabase/blob/chore/settings-ui/apps/studio/components/interfaces/Auth/Hooks/CreateHookSheet.tsx#L331)

Reply:
is the end goal such that the dashboard just uses this FormFieldWrapper instead of FormField_Shadcn_ directly?